.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb", "plugin:react/recommended"],
+    "plugins": ["react"],
+    "rules": {
+      "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+      "import/no-unresolved": "off",
+      "no-shadow": "off"
```

Comment:
Let's add the rule ´"arrow-parens": "off"´ as this has been discussed to cause an issue where the error does not show on the local environment but it does show on the stickler.